frontend/src/components/utils/Gas.tsx

```diff
@@ -5,12 +5,15 @@ interface Props {
 }
 
 const MGAS = new BN(10 ** 6);
+const GGAS = new BN(1024).mul(MGAS);
```

Comment:
Why 1024?

I like the idea of defining the next const using the previous one multiplied by 1000.